.prettierrc

```diff
@@ -24,6 +24,13 @@
       "options": {
         "useTabs": false
       }
+    },
+    {
+      "files": ["apps/docs-snippets2/src/**/*.ts"],
+      "excludeFiles": ["apps/docs-snippets2/src/**/*.test.ts"],
+      "options": {
+        "printWidth": 80
+      }
     }
```

Comment:
By overriding the prettier `printWidth` option to a smaller value for the new doc snippets, we can have code being displayed in a more natural way

## Before
<img width="742" alt="Screenshot 2024-10-02 at 13 00 07" src="https://github.com/user-attachments/assets/0b41342d-edf8-4077-bf4d-d3818083cb33">

## After
<img width="743" alt="Screenshot 2024-10-02 at 13 01 07" src="https://github.com/user-attachments/assets/331b0496-6209-4d3e-bc46-078c88fa4a3b">

Reply:
As far as I'm concerned, we could use 80 across the whole code base. 👀

Reply:
🔥 What a great idea @Torres-ssf